src/app/app.module.ts

```diff
@@ -1,3 +1,4 @@
+import { DirectivesModule } from './../directives/directives.module';
```

Comment:
Not sure how I feel about importing these items as `*Module`, generally the things you import are modules anyway, adding 'Module' to the end of it seems superfluous

Reply:
hym, I guess it's just a convention. If you see other Modules also has a `Module` suffix